client/src/components/judge/Ratings.tsx

```diff
@@ -77,19 +78,84 @@ const Ratings = (props: RatingsProps) => {
-                        }}
-                        className="w-full"
-                    />
+                    <div className="pb-4 pt-2">
+                        {/* React Range Slider */}
+                        <Range
+                            label="Select your value"
+                            step={1}
+                            min={0}
+                            max={10}
+                            values={[categoryScores[i]]}
+                            onChange={(values) => {
+                                const newScores = [...categoryScores];
+                                newScores[i] = values[0];
+                                setCategoryScores(newScores);
+                                if (props.update) submit(newScores);
+                            }}
+                            renderMark={({ props, index }) => (
+                                <div
+                                    {...props}
+                                    key={props.key}
+                                    style={{
+                                        ...props.style,
+                                        height: '4px',
+                                        width: '4px',
+                                        opacity: index == 0 || index == 10 ? 0 : 1,
+                                        backgroundColor:
+                                            index < categoryScores[i] ? '#00ACE6' : '#646464',
+
+                                        borderRadius: '50%',
+                                        marginTop: '1px',
+                                    }}
+                                />
+                            )}
+                            renderTrack={({ props, children }) => {
+                                return (
+                                    <div
+                                        onMouseDown={props.onMouseDown}
+                                        onTouchStart={props.onTouchStart}
+                                        style={{
+                                            ...props.style,
+                                            height: '10px',
+                                            display: 'flex',
+                                            width: '100%',
+                                        }}
+                                    >
+                                        <div
+                                            ref={props.ref}
+                                            style={{
+                                                height: '6px',
+                                                width: '100%',
+                                                borderRadius: '5px',
+                                                background: getTrackBackground({
+                                                    values: [categoryScores[i]],
+                                                    colors: ['#00ACE6', '#ccc'],
```

Comment:
Should use tailwind colors instead of hardcoding the color values. See https://tailwindcss.com/docs/configuration#referencing-in-java-script

Also the slider track background should be `lightest` and the dots should be `light` (see `tailwind.config.js`).